sheets.js

```diff
@@ -1,5 +1,5 @@
-const google = require('googleapis')
-const config = require('config')
+const {google} = require('googleapis')
+const config = require('./config/production.json')
```

Comment:
Since you are using the npm `config`, you don't need to hard-code the path to `production.json`;  You can use the `config` packages env variables to set the path to your JSON.  This has the benefit of supporting multiple deployments (serviced by different configs) without changing code.

Reply:
I'll be the first to admit I have no idea what to do with that - I was just trying to shim in a fix. I don't have bandwidth to make that change at this time.

Reply:
Ok, I can try to get to it; all that needs to be done is reverting the change to line 2.